specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
+Legacy Black            | Legacy       | 900       | 5         | 0      | Legacy        | 900   | 100   | 0     | 0
+Legacy Soft             | Legacy Soft  | 400       | 5         | 0      | Legacy        | 400   | 100   | 0     | 0
+Legacy Soft Bold        | Legacy Soft  | 700       | 5         | 0      | Legacy        | 700   | 100   | 0     | 0
+Legacy Soft Black       | Legacy Soft  | 900       | 5         | 0      | Legacy        | 900   | 100   | 0     | 0
+
+The "Legacy" typographic family has three weights and each weight has regular and "Soft" variants.
+If these were new fonts, they could be implemented as declaring a SOFT design axis. However, these fonts
+predate OpenType 1.8, so their only design axes are those derived from weight, stretch, and style. For
+each weight, this font family has two fonts with identical axis values, so it is not possible to 
+unambiguously select a font in this family using axis values alone.
+
+The hybrid font selection model mitigates this by allowing the specified family name to be _either_
+a typographic family name or a WWS family name. (It can also be an RBIZ name or full name.) In the
+example above, an application can avoid ambiguity by specifying "Legacy" or "Legacy Soft" as the
+family name. These two cases are handled as follows:
+
+  - "Legacy Soft" does not match a typographic family name, so only fonts in the WWS family
+    are considered. These can be differentiated by weight, so there is no ambiguity.
+
+  - "Legacy" matches both a typographic family and a WWS family. All fonts in the typographic
+    family are considered, but presence in the WWS family is used as a tie-breaker. For example,
+    Legacy Bold has the same weight as Legacy Soft Bold, but the former is preferred because
+    it is in the "Legacy" WWS family.
+
+Document and application compatibility issues are mitigated by allowing different types of font
+names to be specified and also by providing a method of deriving axis values from weight, stretch,
+and style parameters.
+
+Suppose an application has family name, weight, stretch, style, and font size parameters. For
+example, these might come from a document's backing store. The application can first pass the
+weight, stretch, style, and font size to the `IDWriteFontSet4::GetDerivedFontAxisValues` method
+to map the input parameters to _derived axis values_. If the application also has explicit axis
+values, it should pass these to `GetDerivedAxisValues` as well. The derived axis values will
+not include any axes there were passed in explicitly. This ensures that explicit axes take
+precedence over derived axis values.
+
+Once the application has axis values (derived, explicit, or both), it can pass the family name
+and axis values to the `IDWriteFontSet4::GetMatchingFonts` method. This method returns an
+`IDWriteFontSet4` representing a list of matching fonts in priority order. The specified family
```

Comment:
Similar to above, it would be useful to know how the priority order is computed or whether there is control over the sorting logic (i.e. priority on a certain dimension/axis?)

Reply:
Maybe I can provide more detail in the spec.